packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -553,10 +567,15 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
    */
   public readonly privateDnsDefault?: boolean = true;
 
-  constructor(name: string, prefix?: string, port?: number) {
-    const region = Lazy.uncachedString({
+  constructor(
+    name: string,
+    prefix?: string,
+    port?: number,
+    { global = false }: InterfaceVpcEndpointAwsServiceProps = {},
```

Comment:
possible to do `props?: InterfaceVpcEndpointAwsServiceProps` ? Easier to read ?

Reply:
Sure, done 👍

Reply:
forget to stage?

Reply:
No, forgot to push this time 😅